src/scripts/main.js

```diff
@@ -1,3 +1,8 @@
 'use strict';
 
-// write your code here
+const tr = [...document.querySelectorAll('tr')];
```

Comment:
You can use just table.rows
![image](https://user-images.githubusercontent.com/98764641/199781521-49b20169-04e6-4be5-b398-32ce594bba24.png)